docs/quickstart.rst

```diff
@@ -1,5 +1,183 @@
-##########
-Quickstart
-##########
+###########
+Quick Start
+###########
+
+This quick start guide means to help the user get FLARE up & running
+quickly without introducing any advanced concepts. For more details, refer
+to :ref:`getting_started`.
+
+Since FLARE offers different modes of running the system, we only cover the simplest approaches here.
+This quick start guide uses the **examples/hello-world/hello-numpy-sag** as an example.
+You will find the details in the example's README.md file.
+
+We also assume you have worked with Python, already set up the virtual env.
+If you are new to this, please refer :ref:`getting_started`.
+
+**Install NVFLARE**
+======================
+
+.. code-block:: shell
+
+  $ python3 -m pip install nvflare
+
+Clone NVFLARE repo to get examples, switch main branch (latest stable branch)
+
+.. code-block:: shell
+
+  $ git clone https://github.com/NVIDIA/NVFlare.git
+  $ cd NVFlare
+  $ git switch main
+
+
+Install requirements/dependencies:
+
+.. code-block:: shell
+
+  $ python3 -m pip install -r examples/hello-world/hello-numpy-sag/requirements.txt
```

Comment:
Do we need this line? There will be no output from pip I think with an empty requirements.txt or will it even run? hello-numpy-sag doesn't seem to have any additional requirements.